ginga/rv/plugins/Catalogs.py

```diff
@@ -8,41 +8,62 @@
 ``Catalogs`` is a local plugin, which means it is associated with a
 channel.  An instance can be opened for each channel.
 
+.. note:: to use ``Catalogs`` it is necessary to install the `astroquery`
```

Comment:
We don't have intersphinx to astroquery, so need double ticks.
```suggestion
.. note:: To use ``Catalogs``, it is necessary to install the ``astroquery``
```